### ISSUE TITLE
[BUG] dropping valid schema fields on bundle

### ISSUE THREAD

**Issue by rlsf** [asyncapi/bundler#185]
### Describe the bug.

in version 2.3.2 there is a regression in the bundle command output.
we use the bundle command to create a single spec file. our spec file adheres to asyncapi spec 2.0.0. and declares it using 
```
asyncapi: 2.0.0
```
as the first line in the doc.
we use tags and custom attributes in our document e.g:
```
asyncapi: 2.0.0
x-company-attr-1: attr-value-1
x-company-attr-2: attr-value-2
info:
  title: Service Name
  version: 1.0.0
  description: Description
  x-company-version: 1.2.3
tags:
  - name: my-tag
    description: tag description
```
executing the bundler results in the custom attributes and the tags being removed. making the doc invalid:
```
asyncapi: 2.0.0
info:
  title: Service Name
  version: 1.0.0
  description: Description
  x-company-version: 1.2.3
```
up until version 2.3.0 it worked as expected

### Expected behavior

preserve custom properties and tags in bundled output.

### Screenshots

see above

### How to Reproduce

use the above snippets and execute bundle command.

### 🥦 Browser

None

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

No, someone else can work on it

**Comment by Souvikns** [comment]
The error is coming from this https://github.com/asyncapi/bundler/blob/a9b0788718705b03013681faea0abb4d80402d60/src/index.ts#L129

**Comment by aeworxet** [comment]
@rlsf 
This bug was addressed in `Bundler` `v0.6.2` that is now included in `CLI` `v2.3.2` as a patch version of `@asyncapi/bundler`: https://github.com/asyncapi/cli/blob/master/package.json#L12

Please do

```bash
rm -rf node_modules
npm install
```

check, and close the issue if the bug is fixed.

**Comment by rlsf** [comment]
verified, works.
